src/WebJobs.Script/ExtensionBundle/ExtensionBundleManager.cs

```diff
@@ -0,0 +1,189 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.IO.Compression;
+using System.Linq;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Microsoft.Azure.WebJobs.Logging;
+using Microsoft.Azure.WebJobs.Script.Configuration;
+using Microsoft.Azure.WebJobs.Script.Properties;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using Newtonsoft.Json;
+using NuGet.Versioning;
+
+namespace Microsoft.Azure.WebJobs.Script.ExtensionBundle
+{
+    public class ExtensionBundleManager : IExtensionBundleManager
+    {
+        private readonly IEnvironment _environment;
+        private readonly ExtensionBundleOptions _options;
+        private readonly ILogger _logger;
+        private readonly string _cdnUri;
+
+        public ExtensionBundleManager(ExtensionBundleOptions options, IEnvironment environment, ILoggerFactory loggerFactory)
+        {
+            _environment = environment ?? throw new ArgumentNullException(nameof(environment));
+            _logger = loggerFactory.CreateLogger(ScriptConstants.LogCategoryExtensionBundles) ?? throw new ArgumentNullException(nameof(loggerFactory));
```

Comment:
change to _logger = loggerFactory.CreateLogger<ExtensionBundleManager>();

Reply:
Adding `loggerFactory.CreateLogger<ExtensionBundleManager>` based on the discussion.